test/travis/check_all_examples_use_standard_init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+FBASE_SNIPPET=test/travis/firebase_init.cc.snip
+FBASE_NOWIFI_SNIPPET=test/travis/no_firebase_init.cc.snip
+for example in `find examples/ -name *.ino`;
+do
+  echo $example;
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_SNIPPET | tr -d '\n'`) ||
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_NOWIFI_SNIPPET | tr -d '\n'`) ||
```

Comment:
Initially I had thought we wanted to compare text->text for diffing but in truth I don't think we do, at least not in the test itself. It would be hard to display because first you would need to decide the closest match in the full source file and then diff against that. This is non-trivial. I think if the test is broken you should just use a visual diffing client (like meld or vimdiff) on your workstation instead of trying to build this into the test.